vtex-apps/search-result
# Add the possibility of usage without search context

#### What is the purpose of this pull request?

Add the possibility of usage without search context (ProductSearchContextProvider)

#### What problem is this solving?

It wasn't possible to use the SearchResult without this context

#### How should this be manually tested?

[Access the workspace](http://pwa--storecomponents.myvtex.com) and see it working. You can also access the storefront to see some changes at the schema.

#### Types of changes
